TeamCode/src/main/java/org/firstinspires/ftc/teamcode/PixyCam.java

```diff
@@ -0,0 +1,170 @@
+package org.firstinspires.ftc.teamcode;
+
+import android.util.Log;
+
+import com.qualcomm.robotcore.hardware.I2cDeviceSynch;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+/**
+ * Created by NoahR on 10/3/17.
```

Comment:
Where did all the following constants come from? I would put the relevant specs links here so other people can double check and/or understand what's going on (I had to read at least http://cmucam.org/projects/cmucam5/wiki/Porting_Guide and http://docs.lpcware.com/lpc800um/RegisterMaps/i2c/r-Register-description.html)